pywwt/quantity_trait.py

```diff
@@ -0,0 +1,12 @@
+from traitlets import TraitType
+from astropy import units as u
+
+class AstropyQuantity(TraitType):
+
+    default = 0 * u.deg
+    info_text = 'Custom trait to handle astropy quantities with units'
+
+    def validate(self, obj, value):
+        if hasattr(value, 'value') and hasattr(value, 'unit'):
```

Comment:
As before, check ``isinstance(value, Quantity)`` instead of checking value and unit separately